examples/console/resources/console/console.deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: console
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: console
+  template:
+    metadata:
+      labels:
+        app: console
+    spec:
+      serviceAccountName: console-server
+      volumes:
+      - emptyDir: {}
+        name: cache
+      containers:
+      ### API
+      - name: console-api
+        image: quay.io/eyefloaters/console-api:0.0.29
+        ports:
+        - containerPort: 8080
+        env:
+        - name: KAFKA_SECURITY_PROTOCOL
+          value: SASL_SSL
+        - name: KAFKA_SASL_MECHANISM
+          value: SCRAM-SHA-512
+        - name: CONSOLE_KAFKA_EXAMPLE
+          value: ${NAMESPACE}/console-kafka
+        - name: CONSOLE_KAFKA_EXAMPLE_BOOTSTRAP_SERVERS
+          value: bootstrap.console-kafka.apps-crc.testing:443
```

Comment:
Code Ready Containers?

Reply:
Good catch, thank you. I've updated this to use a parameter, matching the bootstrap host in `resources/kafka/console-kafka.kafka.yaml`.